src/Module/RemoteFollow.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\DI;
+use Friendica\Core\Logger;
+use Friendica\Core\Protocol;
+use Friendica\Core\Renderer;
+use Friendica\Core\Search;
+use Friendica\Core\System;
+use Friendica\Model\Profile;
+use Friendica\Network\Probe;
+
+/**
+ * Remotely follow the account on this system by the provided account
+ */
+class RemoteFollow extends BaseModule
+{
+	public static function init(array $parameters = [])
+	{
+		if (empty($parameters['profile'])) {
+			return;
+		}
+
+		Profile::load(DI::app(), $parameters['profile']);
+	}
+
+	public static function post(array $parameters = [])
+	{
+		$a = DI::app();
+
+		if (empty($parameters['profile']) || !empty($_POST['cancel']) || empty($_POST['dfrn_url'])) {
+			DI::baseUrl()->redirect();
+		}
+	
+		if (empty($a->profile['uid'])) {
+			notice(DI::l10n()->t('Profile unavailable.') . EOL);
```

Comment:
Please remove ` . EOL` from `notice()` calls, each message is presented separately anyway, so a new line isn't warranted and isn't displayed either.

Reply:
Yeah, this had been left over from C&P from the old place.